docs/rules/0215/foreign-type-reference.md

```diff
@@ -47,8 +47,18 @@ message SomeMessage {
 
 ## Known issues
 
-This package disallows subpackaging, and allows any package that ends with `.type` to
-avoid flagging possible legitimate uses of AIP-213 component packages.
+This check only allows subpackage usage if you're within a versioned path, but generates warnings for unversioned subpackage usage.
+It also ignores if the referenced package is a "common" package like google.api, or if the package path ends in ".type" indicating
+the package is an AIP-213 component package.
+
+Examples of foreign type references and their expected results:
+
+| Calling Package | Referenced Package | Result       |
```

Comment:
I'll be honest, IDK how this will render on the linter site, but let's find out!